### PR TITLE
Update docs as JobManagedBy graduates to Beta

### PR DESCRIPTION
**Pull request by mimowo**
### Description

This is a placeholder PR to update docs as JobManagedBy graduates to Beta. The code PR is already merged https://github.com/kubernetes/kubernetes/pull/127402.

### Issue

Part of https://github.com/kubernetes/enhancements/issues/4368

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] Update docs as JobManagedBy graduates to Beta

**Comment by sftim** [comment]
Once ready for review, please mark this as not-WIP.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/job-managed-by.md [hunk 1]
```diff
@@ -10,5 +10,8 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.30"
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/job-managed-by.md:12]: toVersion: "1.31"

Comment (mimowo) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/job-managed-by.md:12]: sure, done

**Comment by tengqm** [comment]
Change looks good, but I think we need to change `content/en/docs/concepts/workloads/controllers/job.md` as well.

**Comment by mimowo** [comment]
Sure, I just pushed today an update `controllers/job.md` that follows from the default change. PTAL.

**Comment by tengqm** [comment]
Docs lgtm.
@mimowo Please provide a link to the upstream source code change, so that reviewers/approvers can see if the change has landed and the hold can be lifted.
/approve
/hold

**Pull request merged**